src/Components/Settings/Settings.js

```diff
@@ -64,8 +65,8 @@ const Settings = () => {
           data-testid='sound'
           value={settings.sound}
         >
-          <option value={'chordCliff'}>Reverb Splash</option>
-          <option value={'gong'}>Balinese Gong</option>
+          <option value={'reverbSplash'}>Reverb Splash</option>
+          <option value={'balineseGong'}>Balinese Gong</option>
```

Comment:
YES!!!!!

Reply:
Thank you so much for normalizing these names,  I should have done this up front and I really appreciate you taking the time to fix it.